notebooks/featurestore/hsfs/basics/feature_exploration.ipynb

```diff
@@ -1,676 +0,0 @@
-{
```

Comment:
why did you remove this notebook?

Reply:
@AAnastasiia11  are you sure you were asked to get feature_exploration to be deleted?

Reply:
@SirOibaf Ok we figured it out, it has to stay there. @AAnastasiia11 will change more stuff in this card so I'll close this PR and she will open a new one.